### ISSUE TITLE
EnumerateInterfaceAddresses Exception

### ISSUE THREAD

**Issue by daholste** [dotnet/coreclr#25091]
Hey .NET Core CLR,

Any idea about the following exception? This happened on a Mac

```
Internal CLR error. (0x80131506)
   at Interop+Sys.EnumerateInterfaceAddresses(IPv4AddressDiscoveredCallback, IPv6AddressDiscoveredCallback, LinkLayerAddressDiscoveredCallback)
   at Interop+Sys.EnumerateInterfaceAddresses(IPv4AddressDiscoveredCallback, IPv6AddressDiscoveredCallback, LinkLayerAddressDiscoveredCallback)
   at System.Net.NetworkInformation.BsdNetworkInterface.GetBsdNetworkInterfaces()
   at System.Net.NetworkInformation.NetworkInterfacePal.GetIsNetworkAvailable()
   at System.Net.NetworkInformation.NetworkInterface.GetIsNetworkAvailable()
```

Out of four runs today (and many in the past) on the Mac of the same code, this only happened one time

Thank you

**Comment by stephentoub** [comment]
Looks like a dup of https://github.com/dotnet/corefx/issues/38441, which is a dup of https://github.com/dotnet/coreclr/issues/25018, which is a dup of https://github.com/dotnet/coreclr/issues/24727, which is a dup of https://github.com/dotnet/cli/issues/11403, which should be fixed for Preview 6 in https://github.com/dotnet/coreclr/pull/24847. :smile:

**Comment by daholste** [comment]
> Looks like a dup of [dotnet/corefx#38441](https://github.com/dotnet/corefx/issues/38441), which is a dup of #25018, which is a dup of #24727, which is a dup of [dotnet/cli#11403](https://github.com/dotnet/cli/issues/11403), which should be fixed for Preview 6 in #24847. 😄

Amazing -- thank you, @stephentoub!
